### DISCUSSION TITLE
What do you want ASAP?

### DISCUSSION THREAD

**Discussion by NaitLee** [NaitLee/Cat-Printer#9] [Polls]
Though I can't promise I will do them quickly, I want to know what do you *really* want to be here.
This is Discussion, you can also suggest other! 😉
The `TODO` file is also helpful for reference.

**Comment by frankenstein91** [comment]
I hope some day we can add my twitchbot (wip) to this project:
https://github.com/frankenstein91/TwitchPrinterBot/blob/master/bot.py

**Reply by NaitLee** [reply]
You can skim `printer.py` for methods in `class PrinterDriver`. I think it's not so confusing.
Ignore `crc8table` and methods starting with underline. For API integration it's sufficient.

In other file, put `printer.py` aside, you just need to do:
```python3
from printer import PrinterDriver
my_printer = PrinterDriver()
```
then do what you like on `my_printer`.

Of course `server.py` is already doing this. That's already an use case 😄
Also don't forget scripts in folder `additional`, if you need the features. They are imported dynamically.

**Reply by NaitLee** [reply]
I see the script wants to print text.
My own preference is to do PF2 font printing. It's in fact not difficult. And resource saving, compared to the image libs.
I will help you once I made it!

**Reply by NaitLee** [reply]
I think it's very easy to do now. See what's up in #29

**Comment by frankenstein91** [comment]
@NaitLee can you hack a way to prevent opening the browser by cli argument? Would really like to see that 🙂

**Reply by NaitLee** [reply]
It's already there: the `-s` parameter on `server.py`
BTW `-a` for listening on all addresses. But do security concerns before using this broadly.

**Comment by erashmika** [comment]
@NaitLee 
Can you create a **DRIVER FOR WINDOWS** that can work well with these mini printers?

Then we can get any print easily without using any 3rd party software. Like a normal printer. (Plug And Play)
Also, we can connect the mini printer to a POS software and print something like a receipt very easily.

I hope you will give a favorable response to my proposal soon. Also, I hope that the driver I mentioned will be introduced in your next update. I appreciate your hard work for this. Thank you very much. ❤️❤️❤️
